src/textual/_ansi_sequences.py

```diff
@@ -1,11 +1,16 @@
 from __future__ import annotations
 
-from typing import Mapping, Tuple
+from typing import Mapping, Tuple, Type
 
 from .keys import Keys
 
+
+class IgnoredSequence:
+    """Class used to mark that a sequence should be ignored."""
+
+
 # Mapping of vt100 escape codes to Keys.
-ANSI_SEQUENCES_KEYS: Mapping[str, Tuple[Keys, ...] | str] = {
+ANSI_SEQUENCES_KEYS: Mapping[str, Tuple[Keys, ...] | str | Type[IgnoredSequence]] = {
```

Comment:
Using a type as a sentinal is unconventional. I would find it a little astonishing reading this.

Suggest we use `None` as a sentinal, or stick with `Keys.Ignore` in the tuple. 

I objected to this:

```python
if keys == (Keys.Ignore,):
```

because I would find it astonishing to treat that as a sentinal, when the tuple is otherwise a sequence of things. But if check for the `Ignore` in the loop itself, there is no special case needed.